lib/flutter_inapp_purchase.dart

```diff
@@ -128,7 +127,6 @@ class FlutterInappPurchase {
   ///
   /// `iOS` also returns non-subscription products.
   Future<List<IAPItem>> getSubscriptions(List<String> skus) async {
-    if (skus == null || skus.contains(null)) return [];
```

Comment:
No use as List<String> can't be null and any element of it can't be null as well